boards/arm/mercury_xu/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2019, Antmicro
+# SPDX-License-Identifier: Apache-2.0
+config BOARD_MERCURY_XU_BOARD
+	bool "Mercury XU Board"
+	depends on SOC_XILINX_ZYNQMP
+	select DISABLE_TCM_ECC
```

Comment:
Unless there is a board-level reason as to why `DISABLE_TCM_ECC` must be disabled, it should be specified in the `defconfig` (and yes, the symbol should have a prompt as @ioannisg pointed out).